SCMS.Services.Api/Services/Processings/StudentGuardians/StudentGuardianProcessingService.cs

```diff
@@ -0,0 +1,48 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) Signature Chess Club & MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using SCMS.Services.Api.Brokers.Loggings;
+using SCMS.Services.Api.Models.Foundations.StudentGuardians;
+using SCMS.Services.Api.Services.Foundations.StudentGuardians;
+
+namespace SCMS.Services.Api.Services.Processings.StudentGuardians
+{
+    public partial class StudentGuardianProcessingService : IStudentGuardianProcessingService
+    {
+        private readonly IStudentGuardianService studentGuardianService;
+        private readonly ILoggingBroker loggingBroker;
+
+        public StudentGuardianProcessingService(
+            IStudentGuardianService studentGuardianService,
+            ILoggingBroker loggingBroker)
+        {
+            this.studentGuardianService = studentGuardianService;
+            this.loggingBroker = loggingBroker;
+        }
+
+        public StudentGuardian VerifyNoPrimaryStudentGuardianExists(Guid studentId, Guid guardianId) =>
```

Comment:
You might not need this `guardianId` here because you are only checking for a status not against a particular guardian.